commands/web/expand-composer-json

```diff
@@ -5,13 +5,14 @@
 ## Description: Add Drupal core and other needed dependencies.
 ## Usage: expand-composer-json [flags] [PROJECT_NAME]
 ## Example: "ddev expand-composer-json ctools"
-## ProjectTypes: drupal8,drupal9,drupal10
+## ProjectTypes: drupal,drupal8,drupal9,drupal10
 ## ExecRaw: true
 
 export _WEB_ROOT=$DDEV_DOCROOT
-[[ $DDEV_PROJECT_TYPE == "drupal10" ]] && export _TARGET_CORE=^10
-[[ $DDEV_PROJECT_TYPE == "drupal9" ]] && export _TARGET_CORE=^9
-[[ $DDEV_PROJECT_TYPE == "drupal8" ]] && export _TARGET_CORE=^8
+[[ $DDEV_PROJECT_TYPE == "drupal" ]] && export DRUPAL_CORE=^10
```

Comment:
The default for `drupal` could be something else also but probably Drupal 10 is good now and then revisit this in https://github.com/ddev/ddev-drupal-contrib/issues/38.